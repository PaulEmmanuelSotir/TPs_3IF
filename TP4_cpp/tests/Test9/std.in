EN_ERROR_MESS
S seg 0 1 2 3
R rzc 0 1 1 0
OI inter s r
S seg 3 1 2 0
SAVE test.xml
CLEAR
LOAD test.xml
LIST
EXIT
