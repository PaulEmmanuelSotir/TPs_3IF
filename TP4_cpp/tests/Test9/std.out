OK
OK
OK
Error: One or more shape name is invalid
Error: Shape name already existing
OK
OK
OK
* seg : { (0, 1); (2, 3) }
* rzc : { (0, 1); (1, 0) }
